### PR TITLE
[Azure Stack] Fix to match documentation

### PR DESCRIPTION
**Pull request by deathly809**
## Description

Renamed a cmdlet, moved another to not be exported, updated parameter with parameter and update type of other (so that we can hide the cmdlet).

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1 [hunk 1]
```diff
@@ -88,7 +88,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     'Start-AzsScaleUnitNode', 'Get-AzsInfrastructureRoleInstance',
     'Repair-AzsScaleUnitNode', 'Stop-AzsInfrastructureRoleInstance',
     'Stop-AzsScaleUnitNode', 'Add-AzsScaleUnitNode',
-    'New-ScaleOutScaleUnitParametersObject', 'New-ScaleOutScaleUnitParametersListObject'
+    'New-AzsScaleUnitNodeObject'
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1]: Can you please edit the release notes as well

Comment (bganapa) [reply src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1]: Ah realize that, relese notes PR is not merged yet. I will take care of it

**Comment by deathly809** [comment]
@cormacpayne  Since you are most familiar.

File: src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsScaleUnitNode.ps1 [hunk 2]
```diff
@@ -26,29 +26,33 @@ Licensed under the MIT License. See License.txt in the project root for license
 function Add-AzsScaleUnitNode
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsScaleUnitNode.ps1]: Do we have test coverage for this

Comment (deathly809) [reply src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsScaleUnitNode.ps1]: lol no.  I don't have a 20K server to test it.  We will test internally in CI and have OEM partners test.

**Comment by bganapa** [comment]
CI Failure is due to static analysis check issues. Retrying

**Pull request merged**